ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1179,7 +1179,7 @@ periodics:
         done
       command:
       - /bin/bash
-      image: registry.svc.ci.openshift.org/ocp/4.7:cli
+      image: image-registry.openshift-image-registry.svc:5000/ocp/4.7:cli
```

Comment:
This is not nice.
The public one looks much nicer.

Reply:
Asking registry team:
https://coreos.slack.com/archives/C013VBYBJQH/p1608755279429100

Reply:
Had to do this like this for now.
/hold cancel

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1911470